tox.ini

```diff
@@ -2,12 +2,7 @@
 minversion = 3.14.0
 envlist =
     fix_lint,
-    py38,
-    py37,
-    py36,
-    py35,
-    py34,
-    py27,
+    py{38,37,36,35,34,27}{,-upper},
```

Comment:
I'd prefer the other way around; default to no constraints, have the lower as extra; also we should add here pypy too 👍